### ISSUE TITLE
Where are Nuclei templates stored?

### ISSUE THREAD

**Issue by ptrk01** [yogeshojha/rengine#329]
I want to exclude  couple of Nuclei templates which returns information I do not need for my hacking practice. Therefore I would like to exclude them from process. But I could not find location where they are stored. Any help?

I think it would be helpful to have a GUI feature in reNgine to select templates which should be used for Nuclei process.

**Comment by DhiyaneshGeek** [comment]
@yogeshojha is it possible to share the directory where nuclei templates are stored ? are those templates gets updated frequently ?

**Comment by yogeshojha** [comment]
This feature has been already released in v0.5

https://rengine.wiki/pentester/scan_engine/#vulnerability_scan

Please follow the instructions here on how to exclude or include templates. 

Example

```
templates: [files/git-core.yaml, cves/CVE-2020-1234]
```

or

```
templates: [files, cves]
```

Yes, nuclei template gets updated frequently. Before vulnerability scan is performed, it will check if any new templates exists, and will auto-download.
